Makefile

```diff
@@ -13,15 +13,15 @@ lint:
 	golint -set_exit_status ./...
 
 test:
-	go test -race ./...
+	go run scripts/test_with_stripe_mock.go -race ./...
 
 vet:
 	go vet ./...
 
 coverage:
 	# go currently cannot create coverage profiles when testing multiple packages, so we test each package
 	# independently. This issue should be fixed in Go 1.10 (https://github.com/golang/go/issues/6909).
-	go list ./... | xargs -n1 -I {} -P 4 go test -covermode=count -coverprofile=../../../{}/profile.coverprofile {}
+	go list ./... | xargs -n1 -I {} -P 4 go run scripts/test_with_stripe_mock.go -covermode=count -coverprofile=../../../{}/profile.coverprofile {}
```

Comment:
This is a little unfortunate because it means a new stripe-mock will get started with every invocation.

I think it's okay for now though because (1) doesn't affect most test runs (only those that use a custom spec) because we fall back on the default stripe-mock in most cases, (2) stripe-mock's start is relatively fast (in my tests the script adds 0.2-03 seconds of overhead), and (3) it should be plausible to tear this out pretty soon. When 1.12 comes out I'd probably suggest we just drop 1.9 and this hack.

(A motivated individual could also fix it now by plumbing some env vars into the matrix so that coverage only runs on >= 1.10, but it doesn't seem worth it right now to me).